### PR TITLE
Metrics and Metadata changes for adding NGINX Agent

### PR DESCRIPTION
**Pull request by nkashiv**
This PR aims to add changes to the metrics and metadata pages in Amplify documentation to include notes for using NGINX Agent

### CODE REVIEW

File: content/metrics-metadata/other-metrics.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ tags: ["docs"]
 docs: "DOCS-975"
 ---
 
+{{< note >}}Monitoring PHP-FPM and MySQL metrics is only supported by the Amplify Agent.{{< /note >}}
```
Comment (achawla2012) [inline content/metrics-metadata/other-metrics.md:10]: I think we should also put language saying sth like at the moment would be supported in future release of nginx-agent

File: content/metrics-metadata/nginx-metrics.md [hunk 2]
```diff
@@ -460,6 +460,8 @@ nginx.http.request.current = requests.current
 
 The NGINX Plus metrics below are collected *per zone*. When configuring a graph using these metrics, please make sure to pick the correct server, upstream, or cache zone. A more granular peer-specific breakdown of the metrics below is currently not supported in NGINX Amplify.
 
+{{< note >}}NGINX Agent does not support reporting the following metrics *per zone* but it can be used to display a cumulative sum of values from each zone.{{< /note >}}
```
Comment (achawla2012) [inline content/metrics-metadata/nginx-metrics.md:463]: Should we provide definition of `Zone`, if not mentioned any where else in docs?

File: content/metrics-metadata/os-metrics.md [hunk 3]
```diff
@@ -211,3 +166,31 @@ docs: "DOCS-974"
   Type:        gauge, percent
   Description: System swap memory statistics, percentage.
   ```
+
+## Agent Metrics
+
```
Comment (Giners) [inline content/metrics-metadata/os-metrics.md:171]: Since we will be deprecating the Amp agent eventually do we even want this?

**Review by Giners — APPROVED** [review]

File: content/metrics-metadata/other-metrics.md (hunk 1 above)
Comment (achawla2012) [reply content/metrics-metadata/other-metrics.md:10]: Synced offline. At the moment, we wont be having such wording.

**Review by achawla2012 — APPROVED** [review]

**Review by defanator — APPROVED** [review]

**Review by travisamartin — APPROVED** [review]

**Pull request merged**